app/scenes/account_settings/account_settings.js

```diff
@@ -0,0 +1,143 @@
+        }
+    });
+});
+
+export default class AccountSettings extends PureComponent {
+    static propTypes = {
+        theme: PropTypes.object.isRequired
+    }
+
+    static navigationProps = {
+        renderLeftComponent: (props, emitter, theme) => {
+            return (
+                <TouchableOpacity
+                    style={{flex: 1, paddingHorizontal: 15, justifyContent: 'center'}}
+                    onPress={props.onNavigateBack}
+                >
+                    <FormattedText
+                        id='admin.select_team.close'
+                        defaultMessage='Close'
+                        style={{color: theme.sidebarHeaderTextColor}}
+                    />
+                </TouchableOpacity>
+            );
+        }
+    };
+
+    // The enabled arg can be removed once all the scenes have been implemented.
+    buildItemRow = (icon, id, defaultMessage, action, separator = true, enabled = false) => {
+        if (!enabled) {
+            return null;
+        }
+
+        const {theme} = this.props;
+        const style = getStyleSheet(theme);
+
+        return (
+            <View style={style.itemWrapper}>
+                <TouchableOpacity
+                    style={style.item}
+                    onPress={action}
+                >
+                    <View style={style.itemLeftIconContainer}>
+                        <Icon
+                            name={icon}
+                            size={18}
+                            style={style.itemLeftIcon}
+                        />
+                    </View>
+                    <FormattedText
+                        id={id}
+                        defaultMessage={defaultMessage}
+                        style={style.itemText}
+                    />
+                    <Icon
+                        name='angle-right'
+                        size={18}
+                        style={style.itemRightIcon}
+                    />
+                </TouchableOpacity>
+                {separator && <View style={style.separator}/>}
+            </View>
+        );
+    }
+
+    render() {
+        const {theme} = this.props;
+        const style = getStyleSheet(theme);
+
+        return (
+            <View style={style.wrapper}>
+                <View style={style.container}>
+                    <View style={style.itemsContainer}>
+                        {this.buildItemRow('gear', 'user.settings.modal.general', 'General', () => true, true, false)}
+                        {this.buildItemRow('lock', 'user.settings.modal.security', 'Security', () => true, true, false)}
+                        {this.buildItemRow('bell', 'user.settings.modal.notifications', 'Notifications', () => true, true, false)}
+                        {this.buildItemRow('mobile', 'user.settings.modal.display', 'Display', () => true, true, false)}
+                        {this.buildItemRow('wrench', 'user.settings.modal.advanced', 'Advanced', () => true, false, false)}
```

Comment:
I wonder if we can do this by having a `renderOptions` method and inside that method we can call the `buildItemRow` function?